roles/container-engine/cri-o/tasks/main.yaml

```diff
@@ -95,6 +95,17 @@
     - ansible_distribution == "Debian"
     - ansible_distribution_version == "10"
 
+- name: Remove dpkg hold
+  dpkg_selections:
+    name: "{{ item | split ('=') | first }}"
+    selection: install
+  when:
+    - ansible_pkg_mgr == 'apt'
+    - not is_ostree
```

Comment:
Why do we need this line(`- not is_ostree`)?

Reply:
@oomichi thanks for your comment
you're right, the  `- not is_ostree` line seems to be useless.
i've fixed it